axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveCreateFromInvoiceServiceImpl.java

```diff
@@ -144,7 +144,7 @@ public Move createMove(Invoice invoice) throws AxelorException {
               isPurchase ? invoice.getOriginDate() : invoice.getInvoiceDate(),
               invoice.getPaymentMode(),
               invoice.getFiscalPosition(),
-              invoice.getBankDetails(),
```

Comment:
You must keep invoice.getBankDetails() and add also invoice.getCompanyBankDetails()